packages/cosmwasm/src/cw1cosmwasmclient.ts

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { Account, BroadcastMode, Coin, GasLimits, GasPrice, OfflineSigner } from "@cosmjs/launchpad";
+
+import { CosmosMsg } from "./cosmosmsg";
+import { CosmWasmFeeTable, ExecuteResult, SigningCosmWasmClient } from "./signingcosmwasmclient";
+
+export interface CanSendResult {
+  readonly can_send: boolean;
+}
```

Comment:
We could internalize this response type and just return a boolean: `  public canSend(msg: CosmosMsg, address = this.senderAddress): Promise<boolean> {`